src/server/src/main/java/io/cassandrareaper/service/ClusterRepairScheduler.java

```diff
@@ -109,10 +114,15 @@ private boolean keyspaceCandidateForRepair(Cluster cluster, String keyspace) {
   private void createRepairSchedule(Cluster cluster, String keyspace, DateTime nextActivationTime) {
     boolean incrementalRepair = context.config.getIncrementalRepair();
 
+    Set<String> blacklistedTables = context.config.getBlacklistTwcsTables()
+        ? findBlacklistedCompactionStrategyTables(context, cluster, keyspace)
+        : Collections.emptySet();
+
     RepairUnit.Builder builder = RepairUnit.builder()
         .clusterName(cluster.getName())
         .keyspaceName(keyspace)
         .incrementalRepair(incrementalRepair)
+        .blacklistedTables(blacklistedTables)
```

Comment:
We may want to blacklist them only if they're not in the explicit list of tables to repair. One could want to selectively repair TWCS/DTCS tables, which could be allowed if said table is passed in the list of tables to repair.

Also, it should not be possible to create a repair that contains both a blacklist and a list of table.
Thus, there's a little more work to do here : 
- if no table has been explicitly passed (repair the whole keyspace), it's ok to set a blacklist
- if tables were passed, remove (or not?) blacklisted tables from the table list instead of passing a blacklist.

We need to agree on wether or not we allow TWCS tables to be repaired if they're passed in the explicit table list (I think we should allow it).

Reply:
Tables that are explicitly listed are not blacklisted.